lib/membrane/core/parent/crash_group.ex

```diff
@@ -15,9 +16,11 @@ defmodule Membrane.Core.Parent.CrashGroup do
           mode: :temporary,
           members: [Membrane.Child.name()],
           detonating?: boolean(),
-          crash_initiator: Membrane.Child.name()
+          crash_initiator: Membrane.Child.name(),
+          reason: :normal | :shutdown | {:shutdown, term()} | term()
         }
 
   @enforce_keys [:name, :mode]
-  defstruct @enforce_keys ++ [members: [], detonating?: false, crash_initiator: nil]
+  defstruct @enforce_keys ++
+              [members: [], detonating?: false, crash_initiator: nil, reason: nil]
```

Comment:
```suggestion
              [members: [], detonating?: false, crash_initiator: nil, crash_reason: nil]
```
Unify name of the field in `%CrashGroup{}` struct with name of the field passed in context to `handle_crash_group_down`